textproc/py-stanfordnlp/Portfile

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                                         1.0
+PortGroup           python                         1.0
+PortGroup           java                           1.0
+PortGroup           github                         1.0
```

Comment:
please don't use so many redundant spaces... why not:
```
PortSystem              1.0
PortGroup               python 1.0
````
as done in (almost) every other Portfile?

Reply:
Done—thanks.